Update tesla_wall_connector.markdown

pearlythepirate opened the pull request:
<!--
  You are amazing! Thanks for contributing to our project!
  Please, DO NOT DELETE ANY TEXT from this template! (unless instructed).

  Before submitting your pull request, please verify that you have chosen the correct target branch,
  and that the PR preview looks fine and does not include unrelated changes.
-->
## Proposed change
Include a list of entries and a note that it is compatible with the Energy dashboard.



## Type of change
<!--
    What types of changes does your PR introduce to our documentation/website?
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR.
-->

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
<!--
    Details are important, and help maintainers processing your PR.
    Please be sure to fill out additional details, if applicable.
-->

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist
<!--
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR. If you're unsure about any of them, don't hesitate to ask.
    We're here to help! This is simply a reminder of what we are going to look
    for before merging your code.
-->

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards


<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

- **Documentation**
	- Updated Tesla Wall Connector integration documentation.
	- Added a new section detailing supported functionality and available entities, including various sensors.
	- Introduced a new section with step-by-step instructions for integrating the Energy dashboard.
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

pearlythepirate committed:
Update tesla_wall_connector.markdown

pearlythepirate committed:
Update tesla_wall_connector.markdown

Fixed nitpicks

frenck commented on source/_integrations/tesla_wall_connector.markdown:
```diff
@@ -21,3 +21,37 @@ ha_integration_type: integration
 
 {% include integrations/config_flow.md %}
+
+## Entities
```
We have a standardized way of documenting this, see: <https://developers.home-assistant.io/docs/core/integration-quality-scale/rules/docs-supported-functions>

Could you please adjust the PR to match this?

frenck commented on source/_integrations/tesla_wall_connector.markdown:
```diff
@@ -21,3 +21,37 @@ ha_integration_type: integration
+
+These are the entities available in the Tesla Wall Connector integration.
+
+| Domain        | Name               | Enabled |
```
We should avoid the use of tables.

frenck commented on source/_integrations/tesla_wall_connector.markdown:
```diff
@@ -21,3 +21,37 @@ ha_integration_type: integration
+
+## Energy dashboard
+ 
+Energy usage can be easily added to the built-in [Energy dashboard](/docs/energy/) using the Energy sensor.
```
```suggestion
Energy usage can be easily added to the built-in [Energy dashboard](/docs/energy/) using the energy sensor.
```

frenck commented on source/_integrations/tesla_wall_connector.markdown:
```diff
@@ -21,3 +21,37 @@ ha_integration_type: integration
+ 
+Energy usage can be easily added to the built-in [Energy dashboard](/docs/energy/) using the Energy sensor.
+
+To add the Tesla Wall Connector to your Energy dashboard:
+1. Navigate to **Settings** > **Dashboards** > **Energy**
+2. In the "Electricity grid" section, click **Add consumption**
+3. Select the Tesla Wall Connector's "Energy" sensor
+4. Click **Save**
```
```suggestion
To add the Tesla Wall Connector to your Energy dashboard:

1. Navigate to **Settings** > **Dashboards** > **Energy**.
2. In the "Electricity grid" section, select **Add consumption**
3. Select the Tesla Wall Connector's "Energy" sensor.
4. Select **Save**
```

frenck replied:
This comment was result, but not addressed. Un-resolving.

pearlythepirate committed:
Merge branch 'current' into current

pearlythepirate committed:
Update tesla_wall_connector.markdown

Updated to match integration quality standards (https://developers.home-assistant.io/docs/core/integration-quality-scale/rules/docs-supported-functions/)

frenck reviewed (dismissed):
One comment wasn't processed. Could you please take a look?

../Frenck

pearlythepirate committed:
Update source/_integrations/tesla_wall_connector.markdown

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

c0ffeeca7 commented on source/_integrations/tesla_wall_connector.markdown:
```diff
@@ -21,3 +21,36 @@ ha_integration_type: integration
+To add the Tesla Wall Connector to your Energy dashboard:
+
+1. Navigate to **Settings** > **Dashboards** > **Energy**.
+2. In the "Electricity grid" section, select **Add consumption**
```
```suggestion
2. In the **Electricity grid** section, select **Add consumption**.
```

c0ffeeca7 commented on source/_integrations/tesla_wall_connector.markdown:
```diff
@@ -21,3 +21,36 @@ ha_integration_type: integration
+1. Navigate to **Settings** > **Dashboards** > **Energy**.
+2. In the "Electricity grid" section, select **Add consumption**
+3. Select the Tesla Wall Connector's "Energy" sensor.
+4. Select **Save**
```
```suggestion
4. Select **Save**.
```

c0ffeeca7 committed:
tiny tweak

c0ffeeca7 reviewed (approved):
Thank you, @pearlythepirate :+1:
